packages/react/src/hooks/NovuProvider.tsx

```diff
@@ -45,18 +49,16 @@ export const NovuProvider = ({
  * This is needed to differentiate between the hooks and components user agents
  * Better to use this internally to avoid confusion.
  */
-export const InternalNovuProvider = ({
-  children,
-  applicationIdentifier,
-  subscriberId,
-  subscriberHash,
-  backendUrl,
-  apiUrl,
-  socketUrl,
-  useCache,
-  subscriber,
-  userAgentType,
-}: NovuProviderProps & { userAgentType: 'components' | 'hooks' }) => {
+export const InternalNovuProvider = (props: NovuProviderProps & { userAgentType: 'components' | 'hooks' }) => {
+  const applicationIdentifier = 'applicationIdentifier' in props ? props.applicationIdentifier : '';
+  const subscriberHash = 'subscriberHash' in props ? props.subscriberHash : '';
+  const backendUrl = 'backendUrl' in props ? props.backendUrl : '';
+  const apiUrl = 'apiUrl' in props ? props.apiUrl : '';
+  const socketUrl = 'socketUrl' in props ? props.socketUrl : '';
+  const useCache = 'useCache' in props ? props.useCache : undefined;
+  const subscriber = 'subscriber' in props ? props.subscriber : undefined;
+  const subscriberId = 'subscriberId' in props ? props.subscriberId : undefined;
```

Comment:
Same here, feels like it was cleaner before

Reply:
it was needed but we cound another solution to fix it.